tools/cmake/mbed_greentea.cmake

```diff
@@ -3,7 +3,7 @@
 
 set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
 
-include(${MBED_PATH}/tools/cmake/app.cmake)
+include(${mbed-os_SOURCE_DIR}/tools/cmake/app.cmake)
```

Comment:
`${CMAKE_CURRENT_LIST_DIR}/app.cmake`?

Reply:
yes, fixed in the later commit. I realized we should use list dir in tools